src/service/caldavService.js

```diff
@@ -3,7 +3,7 @@
  * SPDX-License-Identifier: AGPL-3.0-or-later
  */
 
-import DavClient from '@nextcloud/cdav-library'
+import { DavClient, CalendarHome, Principal, Collection } from '@nextcloud/cdav-library'
```

Comment:
How to allow imports just for jsdoc? In calendar, the linter always complains about superfluous imports if they are only used inside jsdoc.

Reply:
It just works over here :see_no_evil:

Reply:
>  In calendar, the linter always complains about superfluous imports if they are only used inside jsdoc.

Indeed jshint is complaining, but not eslint ;)